app/scripts/modules/core/src/pipeline/config/actions/history/showHistory.less

```diff
@@ -1,53 +1,30 @@
 @import (reference) '~core/presentation/less/imports/commonImports.less';
 @import (reference) '~bootstrap/less/variables.less';
 
-.modal-show-history {
-  .modal-history-header {
-    padding-bottom: 10px;
-    .middle {
-      padding-left: 5px;
-      padding-right: 5px;
-    }
-  }
+.history-header {
+  padding-bottom: 10px;
+}
 
-  .show-history {
-    position: relative;
-    .form-control {
-      border: 0;
-    }
-    .form-group {
-      margin: 0 -15px;
-    }
-    .summary-nav {
-      cursor: pointer;
-      z-index: 2;
-      width: 12px;
-      position: absolute;
-      top: 0;
-      right: -7px;
-      height: calc(~'100% - 8px');
-      .delta {
-        width: 100%;
-        position: absolute;
-        min-height: 3px;
-        &.add {
-          background: var(--color-success);
-        }
-        &.remove {
-          background: var(--color-danger);
-        }
-      }
-    }
+.show-history {
+  overflow-x: hidden;
+  overflow-y: auto;
+  clear: both;
+  max-height: calc(~'100vh - 220px');
```

Comment:
are most of these styles just copied from what was there before? curious where this 220px comes from?

Reply:
I used they same css from the wizard modal